File: DSA/DivyanshKumarSingh_590012175/Day_16_Question_2.py
def countFrequency(arr, target):
    count = 0
    for num in arr:
        if num == target:
            count += 1
    return count

inp = input("Enter integers separated by spaces: ")
arr = list(map(int, inp.strip().split()))

target = int(input("Enter the target value to count: "))

freq = countFrequency(arr, target)

print(f"The target {target} appears {freq} time(s) in the array.")

File: DSA/DivyanshKumarSingh_590012175/Day_16_Question_2.c
#include <stdio.h>

int countFrequency(int arr[], int size, int target) 
{
    int count = 0;

    for (int i = 0; i < size; i++) 
    {
        if (arr[i] == target) 
        {
            count++;
        }
    }

    return count;
}

int main() 
{
    int n, target;

    printf("Enter number of elements: ");
    scanf("%d", &n);

    int arr[n];

    printf("Enter %d integers: ", n);
    for (int i = 0; i < n; i++) 
    {
        scanf("%d", &arr[i]);
    }

    printf("Enter the target value to count: ");
    scanf("%d", &target);

    int frequency = countFrequency(arr, n, target);

    printf("The target %d appears %d time(s) in the array.\n", target, frequency);

    return 0;
}
